src/components/pages/order/Main/MainRightSide/EmptyMenuAdmin.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react'
+import { styled } from 'styled-components'
+import { theme } from '../../../../../theme';
+import PrimaryButton from '../../../../reusable-ui/PrimaryButton';
+import UserContext from '../../../../../context/UserContext';
+
+export default function EmptyMenuAdmin() {
+  const { resetMenu } = useContext(UserContext)
+    return (
```

Comment:
```suggestion
    
    return (
```

Pense à ajouter un saut de ligne pour aérer le code dans ton composant pour distinguer les parties state / (comportements) / affichage.